configure.ac

```diff
@@ -18,7 +18,7 @@ AC_PREREQ(2.60)
 
 define([VERSION_MAJOR], [1])
 define([VERSION_MINOR], [1])
-define([VERSION_FIX], [0])
+define([VERSION_FIX], [1])
```

Comment:
Please do not increase version within this PR.

New ovirt-web-ui version will be released independently.